adapter/internal/synchronizer/apis_fetcher.go

```diff
@@ -55,24 +55,27 @@ func PushAPIProjects(payload []byte, environments []string) error {
 		logger.LoggerSync.Errorf("Error occured while unzipping the apictl project. Error: %v", err.Error())
 		return err
 	}
-	logger.LoggerSync.Infof("Start Deploying %d API/s...", len(zipReader.File)-1)
 
-	// apiFiles represents zipped API files fetched from API Manager
-	apiFiles := make(map[string]*zip.File, len(zipReader.File)-1)
 	// Read deployments from deployment.json file
 	deploymentDescriptor, envProps, err := sync.ReadRootFiles(zipReader)
 	if err != nil {
 		logger.LoggerSync.Error("Error occured while reading root files ", err)
 		return err
 	}
 
+	numberOfAPIDeployments := len(deploymentDescriptor.Data.Deployments)
```

Comment:
What is the content we get in the deployment descriptor? Can you paste the content here?

Also let's test the following flows.

1. With APIM and CC - API deployment and startup with APIs
2. CC only with apictl

Reply:
content of deployment descriptor looks like below when there are two apis 
![image](https://user-images.githubusercontent.com/22471342/160062395-4587939f-c5c1-43c4-b32c-545e8ee60db2.png)

Reply:
1. CC only with apictl did not log API count information
![image](https://user-images.githubusercontent.com/22471342/160071724-6cbfb556-ecd7-4eb4-85f1-e441c93efcd0.png)

2. With APIM and CC gives the API counts
when deploying 
![image](https://user-images.githubusercontent.com/22471342/160073550-f415956b-cf0d-4826-8eee-872914d3b615.png)

when starting up with two APIs available
![image](https://user-images.githubusercontent.com/22471342/160073931-89aa6b82-6703-47a3-8555-8133f96ea620.png)